terraform/projects/app-ecs-instances/main.tf

```diff
@@ -184,13 +185,16 @@ module "ecs_instance" {
   )}"
 }
 
+
+
+
 resource "aws_ebs_volume" "prometheus_ebs_volume" {
   availability_zone = "${element(data.terraform_remote_state.infra_networking.az_names, 0)}"
   size              = 500
   type              = "gp2"
 
   lifecycle {
-    prevent_destroy = true
+    prevent_destroy = false
```

Comment:
changing this so you can rebuild in your dev environment is fine but this should be changed back for prod

Reply:
changed back to true :D